parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetReader.java

```diff
@@ -52,7 +53,7 @@
  * Read records from a Parquet file.
  * TODO: too many constructors (https://issues.apache.org/jira/browse/PARQUET-39)
  */
-public class ParquetReader<T> implements Closeable {
+public class ParquetReader<T> implements Closeable, VectorizedReader<T> {
```

Comment:
Since now `VectorizedReader` extend `Closeable`, we can remove `Closeable` from here.